.github/workflows/build-wheel-linux-x86_64.yaml

```diff
@@ -5,39 +5,39 @@ on:
     - cron: "0 0 * * 0"
   workflow_dispatch:
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
Just curious about how this works, what is the value here of group (is it unique, shared with other instances, ...)? :)

Reply:
The group is technically how `concurrency` ensures that all workflows/jobs of the same group get queued and run concurrently. Here, we use unique names for each workflow following [the recommended practice in GH docs](https://docs.github.com/en/actions/using-jobs/using-concurrency#example-only-cancel-in-progress-jobs-or-runs-for-the-current-workflow).